aict_tools/io.py

```diff
@@ -30,10 +30,12 @@ def write_hdf(data, path, table_name, mode='w', **kwargs):
 
 
 def get_number_of_rows_in_table(path, key):
-
     with h5py.File(path, 'r') as f:
-        group = f.get(key)
-        return group[next(iter(group.keys()))].shape[0]
+        element = f.get(key)
+        if isinstance(element, h5py._hl.group.Group):
```

Comment:
_hl is a private namespace, `Group` is publicly available just from h5py: `h5py.Group`